random_test.py

```diff
@@ -0,0 +1,281 @@
+"""
+Generate random sylt tests.
+
+The reason why statements can have a size bigger than 1 is that sylt compiles
+larger expressions to multiple assignments, and lua has a hard limit of 200
+local variables, which we want to avoid.
+
+A function is not really a statement, but it makes more sense in this
+abstraction.
+
+Future improvements:
+    - Guarantee longer dependency chains by not just randomly choosing
+      a function/value
+"""
+
+
+from random import choice, random, shuffle
+from typing import Union
+
+indented = lambda s: "\n".join(map(lambda ss: "  " + ss, s.splitlines())) + "\n"  # noqa
```

Comment:
If you want to add a newline to the end, you can put it at the end of the string concatination no?
```suggestion
indented = lambda s: map(lambda ss: f"  {ss}\n", s.splitlines()))
```
(I didn't try this)